specification/network/resource-manager/Microsoft.Network/stable/2024-07-01/loadBalancer.json

```diff
@@ -2840,8 +2840,8 @@
           "description": "The IP address belonging to the backend address."
         },
         "networkInterfaceIPConfigurationId": {
-          "$ref": "./networkInterface.json#/definitions/NetworkInterfaceIPConfiguration",
-          "description": "The id of the network interface ip configuration belonging to the backend address"
+          "type": "string",
+          "description": "Resource ID of the Network Interface IP Configuration."
```

Comment:
>Resource ID [](http://example.com/codeflow?start=26&length=11)

Excellent improvement, thank-you. Also we prefer that properties containing a resource ID include ResourceId in the name (although it's not required and this property name is already getting long). Your choice whether to change the property name here, but consider that feedback going forward.

Reply:
I see, will leave as is due to it being already long, but I'll definitely keep this in mind in the future. Thanks :)